scrape/scrape.go

```diff
@@ -509,6 +514,21 @@ func (sp *scrapePool) sync(targets []*Target) {
 		hash := t.hash()
 
 		if _, ok := sp.activeTargets[hash]; !ok {
+			// The scrape interval and timeout labels are set to the config's values initially,
+			// so whether changed via relabeling or not, they'll exist and hold the correct values
+			// for every target.
+			intervalLabel := t.labels.Get(model.ScrapeIntervalLabel)
+			interval, err := time.ParseDuration(intervalLabel)
+			if err != nil {
+				level.Error(sp.logger).Log("msg", "Error parsing interval label", "err", err, "value", intervalLabel)
```

Comment:
should't we call the setForcedError instead, so the target is not scraped?

Reply:
Yeah, probably.

Reply:
Actually, `setForcedError` is called on a scrape loop, which doesn't exist yet and won't exist because of the `continue`. Also, there won't be any errors here anyway, the labels have already been parsed and checked, the only reason to parse them again here is to use the durations.

Reply:
I guess we could save the durations we parse in `populateLabels`, because we're just repeating the same here, but that would require additional fields (which actually might have to exist anyway for the UI).

Reply:
The goal is to go with the default scrape duration and start the for loop with the forced error.

Reply:
Ok, so if an error is returned from parsing (which it should never because it's already been parsed), we should fall back on the default scrape interval and duration, and set a forced error. Correct?

Reply:
Yes, so it is like if you set a wrong __metrics_path__ or __address__, creates up = 0